media/css/protocol/protocol.scss

```diff
@@ -18,7 +18,7 @@ $image-path: '/media/protocol/img';
 // Already fixed upstream in https://github.com/mozilla/protocol/commit/548e7ce3070372e7fe92f268fb3847e911d5038e
 @media #{$mq-sm} {
     #outer-wrapper .mzp-c-notification-bar {
-        margin: $layout-xs auto 0;
+        margin: $layout-xs auto $spacing-xl;
```

Comment:
Whilst this works it also applies the same bottom margin to every notification on the website, which is something we probably don't want at this time. My suggestion would be to update this selector:

https://github.com/mozilla/bedrock/blob/master/media/css/legal/legal.scss#L70-L72

Note: you may need to increase the specificity to get the changes to apply.